lisp/ess-tracebug.el

```diff
@@ -1227,28 +1227,30 @@ value from EXPR and then sent to the subprocess."
    ((string= el "t") t)
    (t el)))
 
-(defun ess-mpi-handle-messages (buf)
+(defun ess--mpi-handle-messages (buf)
```

Comment:
Can you leave this name unchanged? If we adopt the convention that internal means non-user visible then everything has to be changed. We are adhering to the narrow meaning of "internal" as "internal details extracted for convenience".